core/sources/domains.move

```diff
@@ -71,6 +71,14 @@ module aptos_names::domains {
         registry: Table<NameRecordKeyV1, NameRecordV1>,
     }
 
+    struct ReverseLookupRegistryV1 has key, store {
+        registry: Table<address, NameRecordKeyV1>
+    }
+
+    struct SetReverseLookupEventsV1 has key, store {
+        set_reverse_lookup_events: event::EventHandle<SetReverseLookupEventV1>,
```

Comment:
Any reason for keeping this separate from ReverseLookupRegistryV1?

Reply:
Separating events from their data is the pattern we went with, so I'm just sticking to that.

Reply:
I'm not sure if this is helpful or just spammy and inefficient storage wise (1 extra resource that contains 1 single event). Why not just add it to ReverseLookupRegistryV1? 🤔

Reply:
Anyway this is likely not a blocker.